sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConfig.scala

```diff
@@ -445,9 +455,29 @@ private object DiagnosticsConfig {
     },
     helpMessage = "Cosmos DB Spark Diagnostics, supported value, 'simple'")
```

Comment:
change the help message to include "feed"

Reply:
Fixed